_posts/2022-06-30-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+We are pleased to announce the upcoming 2022 Trino Summit. The summit is
+scheduled as *hybrid* event on the 10th of November 2022. You will be able to
+join us online, or you can make the trip to San Francisco and meet us at the
+Commonwealth Club on the downtown waterfront. Please be aware that you need to
+register regardless of how you'll be joining us.
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://www.starburst.io/info/trinosummit/">
+        Register for the event
+    </a>
+    <a class="btn btn-orange" href="https://sessionize.com/trino-summit-2022/">
+        Sign up to speak
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Starburst is the lead sponsor for the summit, but they're happy to hear from and
```

Comment:
Sounds a bit weird... maybe

but we welcome other sponsor help us make this a great event for the Trinio community.

Reply:
There's no "we" because this is a Trino blog, so it's from the perspective of the Trino community, not from Starburst. Reworded a little, though.